packages/console/src/components/Card/index.tsx

```diff
@@ -1,15 +1,22 @@
 import classNames from 'classnames';
-import React, { ReactNode } from 'react';
+import React, { forwardRef, ReactNode } from 'react';
 
 import * as styles from './index.module.scss';
 
 type Props = {
   children: ReactNode;
+  // eslint-disable-next-line react/require-default-props
   className?: string;
 };
 
-const Card = ({ children, className }: Props) => {
-  return <div className={classNames(styles.card, className)}>{children}</div>;
-};
+const Card = forwardRef<HTMLDivElement, Props>((props, reference) => {
```

Comment:
I don't think is a good idea to change a component by wrapping `forwardRef`, we can use something like `cloneElement` outside of this component, an example is "ActionMenuButton", but the implementation there is not perfect. Not a blocker though, we can have a research or discussion.

Reply:
cc: @simeng-li

Reply:
I need the forwardRef as I need to use `scrollIntoView` in my steps, after expanding a step. So I have to keep the references to those Card elements in order to trigger the scrollIntoView.
How would clone element help in this case?

Reply:
```ts
const CardWithRef = React.forwardRef(
  ({children, ...props}, ref) => {
    return React.cloneElement(children, {...props, ref: anchorRef})
  }
)
```

We can continue researching, is not a blocker, we can optimize it later.

Reply:
Decide not to do this as I've also seen many other components using `forwardRef`, and using cloneElement in the page doesn't look neat to me.